packages/@react-spectrum/list/docs/ListView.mdx

```diff
@@ -361,24 +362,124 @@ This behavior is slightly different in the highlight selection style, where sing
 </Flex>
 ```
 
-{/*
-
 ## Drag and drop
 
-To enable drag and drop in a ListView, you must provide the respective drag and drop hooks sourced from `useDnDHooks` to the ListView's `dragHooks` and `dropHooks` props respectively.
-See the examples below for various common drag and drop use cases. For more information on the hooks themselves and the various supported ways to perform a drag and drop interaction,
+To enable drag and drop in a ListView, you must provide the drag and drop hooks sourced from <TypeLink links={dndDocs.links} type={dndDocs.exports.useDragAndDrop} /> to the ListView's `dragAndDropHooks` prop.
+See the examples below for various common drag and drop use cases. For more information on `useDragAndDrop` and the various supported ways to perform a drag and drop interaction,
 please see the [drag and drop documentation](dnd.html).
 
+### Draggable ListView and droppable ListView
+
 The example below demonstrates how to create a draggable ListView and a droppable ListView.
 
 <details>
   <summary style={{fontWeight: 'bold'}}><ChevronRight size="S" /> Show code</summary>
 
 ```tsx example export=true render=false
-import {useDnDHooks} from '@react-spectrum/dnd';
+import {useDragAndDrop} from '@react-spectrum/dnd';
 import {useListData} from '@react-stately/data';
 
-function DragIntoListExample() {
+function DraggableListView(props) {
```

Comment:
Each example currently has its own implementation of a DraggableListView and a DroppableListView. I could instead make a single generic ListView that accepts drag/drop hook options via props and have the examples all reuse that component but I felt it would be clearer to have the drag/drop hook options provided inline to the `useDragAndDrop` hook. The downside to this is that there is a lot of duplicate code but it also means the user can copy paste from which ever example's code block they are interested in and not have to worry about finding the base component implementation hidden somewhere above

Lemme know what you think. If we wanna go with the single generic ListView component that accepts the DnDOptions then the first example's setup can be used as a starting point

Reply:
Yeah I think it's ok since they are collapsed. We can see what people think.